static/styles/settings.css

```diff
@@ -322,7 +322,7 @@ td .button {
 
             &[data-sort]:hover {
                 cursor: pointer;
-                background-color: hsla(0, 0%, 95%);
+                background-color: hsl(211, 29%, 14%);
```

Comment:
Can you explain what this change does?  It seems like the sort of thing that will work in one theme but not the other; do you mean to change this in `dark_mode.css`?.

Reply:
Actually, this change is not required. I've removed this as adding the class `table-sticky-headers` is sufficient to solve this bug.
We also have a similar bug for the `alert-word` settings table as well, so I've pushed the change for that as well.